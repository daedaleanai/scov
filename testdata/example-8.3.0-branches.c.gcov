version:8.3.0
file:example.c
function:28,59,1,main
lcount:28,1,0
lcount:34,1,0
branch:34,taken
branch:34,nottaken
lcount:36,1,0
lcount:37,1,0
lcount:43,1,0
lcount:44,1,0
lcount:49,1,0
branch:49,nottaken
branch:49,taken
lcount:51,0,1
lcount:55,1,0
lcount:58,1,0
