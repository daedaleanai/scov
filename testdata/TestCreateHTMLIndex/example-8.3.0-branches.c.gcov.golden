<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>SCov</title>
<meta name="description" content="Code coverage report">
<link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
<!--[if lte IE 8]>
	<link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-old-ie-min.css">
<![endif]-->
<!--[if gt IE 8]><!-->
	<link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-min.css">
<!--<![endif]-->
<link rel="generator" href="https://gitlab.com/stone.code/scov">
<style>
html { padding:1em; }
body { max-width:70em; margin:auto; }
table { margin-bottom: 1em; }
.coverage { min-width:100%; }
.coverage td:nth-child(2), .coverage th:nth-child(2) { text-align:center; }
.coverage td:nth-child(3), .coverage th:nth-child(3) { text-align:center; }
.coverage td:nth-child(4), .coverage th:nth-child(4) { text-align:center; }
.sparkbar { border: 1px solid black; border-radius:1px; min-width:50px; height:1em; }
.sparkbar .fill { display: inline-block; height: 100%; }
.sparkbar .high { background-color:lightgreen; }
.sparkbar .medium { background-color:yellow; }
.sparkbar .low { background-color:red; }
.sparkbar .empty { display: inline-block; height: 1em; background-color: white; }
footer { border-top: 1px solid rgb(203, 203, 203); margin-top: 1em; background: #e0e0e0; padding: .5em 1em; }
@media screen and (min-width: 48em) {
	.pure-gutter-md > div { box-sizing: border-box; padding: 0 0.5em; }
	.pure-gutter-md > div:first-child { padding-left: 0; }
	.pure-gutter-md > div:last-child { padding-right: 0; }
}
@media screen and (max-width: 48em) {
	.table-md td, .table-md th { padding: 0.5em; }
}
</style>
</head>
<body>
<div class="pure-g"><h1 class="pure-u">SCov</h1></div>
<div class="pure-g pure-gutter-md"><div class="pure-u-1 pure-u-md-1-2">
<h2>Metadata</h2>
<p>Date: Mon Jan  2 15:04:05 UTC 2006</p>
</div><div class="pure-u-1 pure-u-md-1-2">
<h2>Coverage Summary</h2>
<table class="pure-table pure-table-horizontal coverage">
<thead><tr><th></th><th>Hit</th><th>Total</th><th>Coverage</th><tr></thead>
<tbody>
<tr><td>Lines:</td><td>9</td><td>10</td><td>90.0%</td></tr>
<tr><td>Functions:</td><td>1</td><td>1</td><td>100.0%</td></tr>
<tr><td>Branches:</td><td>2</td><td>4</td><td>50.0%</td></tr>
</tbody>
</table></div></div>
<div class="pure-g"><div class="pure-u-1">
<h2>By File</h2>
<table class="pure-table pure-table-bordered table-md" style="width:100%">
<thead><tr><th>Filename</th><th colspan="3">Line Coverage</th><th colspan="3">Function Coverage</th><th colspan="3">Branch Coverage</th></tr></thead>
<tbody>
<tr><td><a href="example.c.html">example.c</a></td><td><div class="sparkbar"><div class="fill high" style="width:90.0%"></div><div class="empty" style="width:10.0%"></div></div></td><td>9/10</td><td>90.0%</td><td><div class="sparkbar"><div class="fill high" style="width:100%"></div></div></td><td>1/1</td><td>100.0%</td><td><div class="sparkbar"><div class="fill low" style="width:50.0%"></div><div class="empty" style="width:50.0%"></div></div></td><td>2/4</td><td>50.0%</td></tr>
</tbody>
</table>
</div></div>
<footer>Generated by <a href="https://gitlab.com/stone.code/scov">SCov</a>.</footer>
</body>
</html>